.github/dependabot.yml

```diff
@@ -9,3 +9,16 @@ updates:
     directory: "/vale"
     schedule:
       interval: daily
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    reviewers:
+      - "buildkite/foundation"
```

Comment:
Maybe onboarding?

Reply:
hah yeah I wondered about this one. Onboarding sounds good 👍